<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gambeer Brewing Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hl_1">
     <item>
      <widget class="QLabel" name="label_temp_inf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Temperature Lower</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTempInf">
       <property name="inputMask">
        <string>000; </string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hl_2">
     <item>
      <widget class="QLabel" name="label_temp_sup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Temperature Higher</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTempSup">
       <property name="inputMask">
        <string>000; </string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hl_3">
     <item>
      <widget class="QLabel" name="label_time">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Time (hh:mm:ss):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTime">
       <property name="inputMask">
        <string>00:00:00; </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_1">
     <item>
      <widget class="QLabel" name="label_temp">
       <property name="text">
        <string>Temperature</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberTemp"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vl_2">
     <item>
      <widget class="QLabel" name="label_time_2">
       <property name="text">
        <string>Time</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumberTime"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOff">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
